### DISCUSSION TITLE
Community Edition: first access and the system doesn't send the first email

### DISCUSSION THREAD

**Discussion by marcomazzaglia** [mozilla/hubs#6481] [Hubs Cloud Questions]
Good morning everyone.
I have a problem with the installation of the Community Edition of Mozilla Hubs.
The installation process with the scripts went through successfully without any errors.
The problem is that I can't get past the first boot. It asks me to enter the administrator's email to send the message with the link to confirm the marked address but then it does not send me the email.
I have entered the address marked as 'Admin user' (or something similar - I don't remember the default field in the configuration file) as my email address but then I don't get any messages either in the box or in SPAM.
To check this, I installed "msmtp" and I tried to configure it with the same data given in the Mozilla Hubs configuration file and there it worked without any problems in the same server.

1. Am I doing something wrong in the procedure when first starting Mozilla Hubs? Is a default admin account perhaps already defined which I have not found in my documentation?
2. Where do I go to check and understand whether the query to the client-side SMTP server actually started the e-mail

The certificates used within Hubs were generated with Let's Encrypt and Mozilla Hubs is self hosted on Kubernetes.

Thanks in advance to all.

**Comment by hrithikwins** [comment]
To find out the cause of the problem you can have a look at the logs of your reticulum when you sign in
`kubectl logs <reticulum-pod-name> -n <namescape-name>`

Most likely cause is your smtp not working, the best way would be to use an online smtp tester to check if you are giving the right config [Online smtp testers](https://letmegooglethat.com/?q=online+smtp+tester) (your answer to Q.2)

Also if you are using a subdomain as your hubs_domain, you need to confirm that your config points the noreply@yourdomain.com instead of noreply@subdomain.yourdomain.com in your hcce.yaml file, in the render_hcce.yaml you define your email is the admin email by default, if you are using SES from AWS as your smtp you would want to make sure that the email is allowed to receive emails from SES (you need to verify SES identity) 

There are lots of discussions on discord for the same 
https://discord.com/channels/498741086295031808/1179844077819269250/1179844235818713179
